pandas/core/series.py

```diff
@@ -3441,8 +3441,10 @@ def sort_values(
                 )
             ascending = ascending[0]
 
-        if not is_bool(ascending):
-            raise ValueError("ascending must be boolean")
+        #if not is_bool(ascending):
```

Comment:
is_bool(ascending) is equivalent to `isinstance(ascending, (bool, np.bool_)`, so this shouldn't make a difference.  if im wrong and this does make a difference, this needs a test

Reply:
i see this issue open #41634. if this is not resolved, then is_bool function is not working as expected.

regarding the comment, wasnt aware of the same as this is my first contribution. will note this for future.

Reply:
@deej197 Isn't this issue for DataFrame not series? 
https://github.com/pandas-dev/pandas/blob/dad3e7fc3a2a75ba5f330899be0639cff0f73f6c/pandas/core/frame.py#L6208

You might want to look into using ``validate_ascending``
https://github.com/pandas-dev/pandas/blob/dad3e7fc3a2a75ba5f330899be0639cff0f73f6c/pandas/util/_validators.py#L421

Reply:
@lithomas1 I can make the changes in this validate_ascending function , but i am stuck in running tests locally. I tried replacing the pandas package in C:\Users\usename\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages with the modified package, but i dont see the changes being applied.

appreciate any suggestions on executing tests internally.